metecho/api/models.py

```diff
@@ -1198,7 +1206,7 @@ class ScratchOrg(
     description = MarkdownField(blank=True, property_suffix="_markdown")
     org_type = StringField(choices=ScratchOrgType.choices)
     org_config_name = StringField()
-    owner = models.ForeignKey(User, on_delete=models.PROTECT)
+    owner = models.ForeignKey(User, on_delete=models.PROTECT, blank=True, null=True)
```

Comment:
Does this require changes to the `save` method below, where we reference `self.owner.id`?

Reply:
I think we can modify change the [conditional](https://github.com/oddbird/Metecho/blob/658e2d5dceb1f3030c26d88e9a25e53a9adbea39/metecho/api/models.py#L1276) to `if is_new and self.owner:`